open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -119,12 +119,17 @@
    "outputs": [],
```

Comment:
<p>I think this function is not a in place process right? So the return value of the line 14 should be assigned to a variable? Or here you just want to run it as a sample?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>The instruction is :  From the names we can deduce that countries like Australia and China have more detailed breakdown by provinces. Let's look for information on China to see the example, So I just want to show the information</p>